projects/plugins/jetpack/extensions/blocks/tiled-gallery/tiled-gallery.php

```diff
@@ -185,3 +205,4 @@ private static function is_squareish_layout( $attr ) {
 }
 
 Tiled_Gallery::register();
+add_action( 'init', array( __NAMESPACE__ . '\Tiled_Gallery', 'inject_javascript_options' ) );
```

Comment:
I think we should also stick to block-specific hooks here (this might trigger for the RSS feed and the API too). Sorry, I know it was my suggestion. 🙃 

```suggestion
add_action( 'enqueue_block_assets', array( __NAMESPACE__ . '\Tiled_Gallery', 'inject_javascript_options' ) );
```

Reply:
Done in 3a4cb109677, thanks!